lib/tmdb/client.rb

```diff
@@ -294,7 +336,7 @@ def searchable_query(query)
         return unless query.present?
         # If a user searches for a name that starts with an `&` the api call fails.
         # This ensures no non alphanumeric characters make it into the query string.
-        I18n.transliterate(query.gsub(/[^0-9a-z ]/i, ''))
+        I18n.transliterate(query).gsub(/[^0-9a-z ]/i, '')
```

Comment:
Fixed this bug. It was previously "fixing" a name like "María Ylara" to "Mara Ylara". Now it is correctly producing "Maria Ylara".